Metric/MetricHandler.php

```diff
@@ -99,7 +99,7 @@ public function setContainer(ContainerInterface $container): void
 
     public function setCurrentRequest(RequestStack $requestStack): void
     {
-        $this->currentRequest = $requestStack->getCurrentRequest();
+        $this->currentRequest = $requestStack->getMasterRequest();
```

Comment:
Souldn't it be called `masterRequest` or `request` into Metric/MetricHandler.php

Reply:
actually, I'm gonna change it to being the request stack and call getMasterRequest as late as possible